app/src/main/java/com/example/next_show/fragments/ShowDetailFragment.java

```diff
@@ -108,7 +108,7 @@ private void setUpView(View currView) {
             btnSaveShow.setVisibility(View.GONE);
 
             // check if user already rated it, show if user liked it or not
-            if (currShow.getUserLiked() != null) {
+            if (currShow.getUserLiked().equals("liked") || currShow.getUserLiked().equals("disliked")) {
```

Comment:
Do you not need to check `null` anymore?

Reply:
The Parse database doesn't accept strings being null so I have to initialize it as a blank string hence the check if the string actually equals a rating